ALLHUBS-Jan-2024-Liftoff/code-stars
# 92 review usernames

In this branch, I:
- Created a one-to-many relationship between users and reviews
- Updated DTOs and controllers to reflect this change
- Updated the review form to submit the id of the current user
- Added usernames to reviews
- Added tags to reviews with custom CSS styling

To test it:
1. Log in to any account
2. Write a review with tags
3. Verify that the account's username and tags display with the review
4. Edit the review, verify changes are made
5. Delete the review, verify it's gone

NOTE: Reviews can no longer be submitted without signing in first!
